src/components/Property/PropertyDetails.js

```diff
@@ -2,52 +2,81 @@ import PropTypes from 'prop-types'
 import LocationAlerts from './LocationAlerts'
 import PersonAlerts from './PersonAlerts'
 import Tenure from './Tenure'
+import Link from 'next/link'
 
 const PropertyDetails = ({
   address,
   hierarchyType,
   locationAlerts,
   personAlerts,
   tenure,
-}) => (
-  <div>
-    <h1 className="govuk-heading-l">
-      {hierarchyType.subTypeDescription}: {address.addressLine}
-    </h1>
-
-    <div className="govuk-grid-row">
-      <div className="govuk-grid-column-one-half">
-        <div className="govuk-body-s">
-          <div className="property-details-main-section">
-            <span className="govuk-body-xs">Property details</span>
-            <br></br>
-            <span className="govuk-!-font-weight-bold text-green">
-              {address.addressLine}
-            </span>
-            <br></br>
-            {address.streetSuffix && (
-              <>
-                <span className="govuk-!-font-weight-bold text-green">
-                  {address.streetSuffix}
-                </span>
-                <br></br>
-              </>
-            )}
-            <span className="govuk-body-xs text-green">
-              {address.postalCode}
-            </span>
+}) => {
+  let repairsStatus = (status, subTypeDescription) => {
+    if (status) {
+      return (
+        <span class="govuk-heading-m">
+          <strong>
+            <Link href="">
+              <a>Raise a repair on this {subTypeDescription.toLowerCase()} </a>
+            </Link>
```

Comment:
Did we say to have a link or just text for now? It might be worth checking with Sarah but it could be a bit misleading to have a link that isn't functional